### ISSUE TITLE
GTK apps missing fonts

### ISSUE THREAD

**Issue by notklaatu** [flatpak/flatpak#5389]
I'm running RHEL 9.1 and it seems that any Flatpak app using GTK is missing its default font, showing only boxes instead of useful glyphs.

Deja Dup:
![deja-flatpak](https://user-images.githubusercontent.com/2211671/232988479-37396e3f-f902-4503-b5ea-e2eaa151e5b8.png)

GIMP: 
![gimp-flatpak](https://user-images.githubusercontent.com/2211671/232988505-79748168-4099-4acf-a7fa-7afb77f059a5.png)

Also, Inkscape, GNOME Boxes, and others.

NOT Krita or Goxel or Steam, by contrast. These apps have access to all the fonts they need.

I did try this:

```bash
$ flatpak repair
```

I've tried uninstalling GIMP and reinstalling, but that didn't fix it.

I assume there's some problem with an installed (or missing?) SDK but I don't know where to begin to investigate further.

**Comment by notklaatu** [comment]
I ran `flatpak run -vv org.gimp.GIMP 2> ~/gimp.err.log` and noticed this in the output:

```
(gimp-2.10:2): Pango-WARNING **: 18:48:57.212: failed to create cairo scaled font, expect ugly output. the offending font is 'Cantarell 11'

(gimp-2.10:2): Pango-WARNING **: 18:48:57.212: font_face status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.212: scaled_font status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.212: failed to create cairo scaled font, expect ugly output. the offending font is 'Cantarell 11'

(gimp-2.10:2): Pango-WARNING **: 18:48:57.212: font_face status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.212: scaled_font status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.213: failed to create cairo scaled font, expect ugly output. the offending font is 'Cantarell 15.83984375'

(gimp-2.10:2): Pango-WARNING **: 18:48:57.213: font_face status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.213: scaled_font status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.213: failed to create cairo scaled font, expect ugly output. the offending font is 'Cantarell 15.83984375'

(gimp-2.10:2): Pango-WARNING **: 18:48:57.213: font_face status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:57.213: scaled_font status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:58.168: failed to create cairo scaled font, expect ugly output. the offending font is 'Cantarell 9.166015625'

(gimp-2.10:2): Pango-WARNING **: 18:48:58.168: font_face status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:58.168: scaled_font status is: file not found

(gimp-2.10:2): Pango-WARNING **: 18:48:58.168: failed to create cairo scaled font, expect ugly output. the offending font is 'Cantarell 9.166015625'
```

**Comment by notklaatu** [comment]
Maybe not an *exact* duplicate. Reporter for #5288 says:

> The strange thing is, if I was to use a default system font like Cantarel the problem goes away. 

I am using Cantarell as my system font, haven't changed as far as I can remember (and anyway not since my Flatpaks were working correctly and now), and the Flatpaks still render as empty boxes. 

For kicks, I ran this command:

```bash
$ flatpak run --command=fc-cache org.gimp.GIMP -f -v /usr/share/fonts/cantarell
fc-cache: ../../src/fccache.c:808: FcCacheFini: Assertion `fgGacheChains[i] == NULL' failed.
```

**Comment by notklaatu** [comment]
The command `flatpak run --command=fc-cache org.gimp.GIMP -f -v /usr/share/fonts/cantarell` fixed the issue.

Exact steps  I took (I can't verify which ones  are necessary, because I did them without testing between each step):

1. Open GNOME Tweaks > Appearance > set font to Cantarell even though font is already Cantarell 
2. Run `flatpak run --command=fc-cache <Flatpak name, such as org.gimp.GIMP or org.gnome.DejaDup> -f -v /usr/share/fonts/cantarell`
3. Ignore fc-cache errors. Apparently they don't matter in this context.
4. Launch the Flatpak app from **Activities**

Do this for each Flatpak affected.
